etcdutl/etcdutl/backup_command.go

```diff
@@ -178,21 +178,10 @@ func saveSnap(lg *zap.Logger, destSnap, srcSnap string, desired *desiredCluster)
 // mustTranslateV2store processes storeData such that they match 'desiredCluster'.
```

Comment:
You are no longer using `storeData`. Please update the comment and function arguments.

Reply:
updated